assets/wizards/setup/components/plugin-link-card/style.scss

```diff
@@ -0,0 +1,50 @@
+.newspack-service-link-card.muriel-component {
+	background: white none center 33px no-repeat;
+	color: black;
+	display: block;
+	padding: 120px 42px 40px 42px;
+	position: relative;
+	text-align: center;
+	text-decoration: none;
+	white-space: normal;
+	width: 100%;
+	&.jetpack,
+	&:active.jetpack,
+	&:focus.jetpack,
+	&:hover.jetpack {
+		background: white url( './jetpack.svg' ) center 33px no-repeat;
+	}
+	&.google-site-kit,
+	&:active.google-site-kit,
+	&:focus.google-site-kit,
+	&:hover.google-site-kit {
+		background: white url( './sitekit.svg' ) center 33px no-repeat;
+	}
+	.components-spinner {
+		display: block;
+		float: none;
+		margin: 10px auto 0 auto;
+	}
+	.checklist__task-icon {
+		background-color: $muriel-green-400;
+		border: 2px solid $muriel-green-400;
+		border-radius: 16px;
+		display: none;
+		height: 16px;
+		right: 26px;
+		position: absolute;
+		top: 14px;
+		width: 16px;
+		svg {
+			width: 16px;
+			height: 16px;
+			color: $muriel-white;
+		}
+	}
+	&.is-configured {
```

Comment:
Suggestion: Can we make it clearer that a plugin has been configured? Adding the word 'Configured' next to the check (and maybe decreasing the opacity further) would help. When I first went through the wizard I assumed the small green check meant the plugin was configured, but I wasn't super sure.

Reply:
Good point @claudiulodro . I've updated the design to this so there is a separate link in each plugin card. This will help indicate that they can be clicked individually: 
<img width="603" alt="Screenshot 2019-06-28 16 12 34" src="https://user-images.githubusercontent.com/4823744/60374001-8cb53580-99bf-11e9-8774-03b9ed75b75f.png">

And this is what it would look like if one of the plugins has been configured: 
<img width="560" alt="Screenshot 2019-06-28 16 01 55" src="https://user-images.githubusercontent.com/4823744/60373965-65f6ff00-99bf-11e9-909e-2808c074a0f2.png">

Reply:
This is done in https://github.com/Automattic/newspack-plugin/pull/114/commits/4e0ac93145eb2b47f0e67ef050532be7d7c03067

Reply:
Looks great! Approved.